src/web-animations-next-player.js

```diff
@@ -30,6 +30,38 @@
 
   // TODO: add a source getter/setter
   scope.Player.prototype = {
+    _updateChildTiming: function(childPlayer, offset) {
+      if (childPlayer.startTime != this.startTime + offset) {
+        if (this.startTime === null) {
```

Comment:
> Maybe while we're moving this we could look at swapping the order of these tests - if this.startTime == null, then this.startTime + offset = offset, which could potentially equal childPlayer.startTime, even if childPlayer.currentTime != this.source.player.currentTime - offset.

Wait, we don't want to alter this.startTime here at all. All _updateChildTiming is doing now is scheduling children relative to the parent (taking into account any delay on the parent, plus arranging children within a sequence), and doing the thing that we don't know the purpose of.

If we can delete the mysterious code I will rename this method to something like "arrangeChildren".